Parquet Designer.csv

```diff
@@ -1,26 +1,3 @@
-ID,Name,Subtype,TrenchAdjective,AddsToDesert,AddsToForest,AddsToHeavens,AddsToVolcanic,AddsToSeaside,AddsToSwamp,AddsToTundra,IsWalkable,IsHole,IsFlammable,IsLiquid,ModTool,GatherTool,MaxToughness
-0,Dirty,Floor,Dirty,TRUE,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,TRUE,FALSE,FALSE,FALSE,Shovel,<null>,<null>
-1,Grass,Floor,Dusty,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,TRUE,FALSE,Shovel,<null>,<null>
-2,Long Grass,Floor,Muddy,FALSE,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,Shovel,<null>,<null>
-3,Sand,Floor,Dry,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,Shovel,<null>,<null>
-4,Snow,Floor,Snowy,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,Shovel,<null>,<null>
-5,Stone,Floor,Rocky,TRUE,FALSE,FALSE,TRUE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,Shovel,<null>,<null>
-6,Plank,Floor,Wooden,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,Saw,<null>,<null>
-7,Tile,Floor,Tiled,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,Saw,<null>,<null>
-8,Water,Block,<null>,FALSE,FALSE,FALSE,FALSE,TRUE,TRUE,FALSE,FALSE,FALSE,FALSE,TRUE,<null>,Bucket,10
-9,Lava,Block,<null>,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,TRUE,<null>,Bucket,10
-10,Tree,Block,<null>,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,<null>,Axe,10
-11,Bush,Block,<null>,FALSE,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,<null>,Axe,10
-12,Rushes,Block,<null>,FALSE,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,<null>,Scythe,10
-13,Sandstone,Block,<null>,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,<null>,Pick,10
-14,Ice,Block,<null>,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,<null>,Pick,10
-15,Boulder,Block,<null>,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,<null>,Pick,10
-16,Board,Block,<null>,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,<null>,Pick,10
-17,Brick,Block,<null>,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,<null>,Pick,10
-18,Chair,Furnishing,<null>,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,<null>,<null>,5
-19,Table,Furnishing,<null>,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,<null>,<null>,5
-20,Door,Furnishing,<null>,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,<null>,<null>,5
-21,Flower,Collectable,<null>,FALSE,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,<null>,<null>,0
-22,Weeds,Collectable,<null>,FALSE,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,TRUE,FALSE,<null>,<null>,0
-23,Bones,Collectable,<null>,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,FALSE,FALSE,FALSE,<null>,<null>,0
-24,Copper Ore,Collectable,<null>,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,<null>,<null>,5
\ No newline at end of file
+ID,Name
```

Comment:
Is this a placeholder file?

Reply:
Yeah, it will be updated with real content.  :)